packages/blocks/framework/src/lib/property/input/common.ts

```diff
@@ -16,6 +16,26 @@ export const SupportsAISchema = Type.Object({
 
 export type SupportsAISchema = Static<typeof SupportsAISchema>;
 
+export const TMandatoryPropertyValue = <
+  T extends TSchema,
+  U extends PropertyType,
+>(
+  T: T,
+  propertyType: U,
+): TObject =>
+  Type.Object({
+    type: Type.Literal(propertyType),
+    required: Type.Literal(true),
+    defaultValue: Type.Optional(Type.Any()),
+  });
+
+export type TMandatoryPropertyValue<T, U extends PropertyType> = TPropertyValue<
+  T,
+  U,
+  ValidationInputType.ANY,
+  true
```

Comment:
re-uses the TPropertyValue defined below and passes the final boolean TRUE for required